### ISSUE TITLE
headers in http plugin not working

### ISSUE THREAD

**Issue by morya** [reddec/monexec#9]
I think this project is quite what I need.
nice one.

but I think headers config in http plugin is not working,
I think, it should be like this,

```golang
func (c *Http) renderAndSend(message string, params map[string]interface{}) {
	/// .....
	if err != nil {
		c.log.Println("failed prepare request:", err)
		return
	}
	
	for k,v := range c.Headers {
		req.Header.Set(k, v)
	}
```

if that's ok, I can send a PR.

**Comment by morya** [comment]
for example, I would like send with-in the headers for POST request

```
Content-Type: application/x-www-form-urlencoded
```

**Comment by reddec** [comment]
fixed in #10  
thanks @morya !
